lib/plugins/aws/deployFunction/index.js

```diff
@@ -62,7 +66,8 @@ class AwsDeployFunction {
     return BbPromise.resolve();
   }
 
-  zipFunction() {
+  packageFunction() {
+    this.serverless.cli.log(`Packaging function: ${this.options.function}...`);
```

Comment:
oh, that's a bummer. just opened a PR to improve to fix the experience on Windows again: https://github.com/serverless/serverless/pull/2714